resources/assets/js/app/App.js

```diff
@@ -13,7 +13,7 @@ class App extends React.Component {
             <div id="application">
                 <MainHeader />
 
-                <div className="main-container container py-4">
+                <div className={this.props.location.pathname !== '/' ? 'main-container container py-4' : ''}>
```

Comment:
create nav helper or nav service with method isRootPath or similar that will know how to define what path will be root, avoid using such checks directly in template